local-config.yml

```diff
@@ -11,7 +11,7 @@ utils:
 test_interval:
 - test: ping
   type: sec
-  duration: 20
+  duration: 14
```

Comment:
Let's drop this change as a part of this PR.

Reply:
ok i will drop this change